supporting_content/dimensions/region.md

```diff
@@ -0,0 +1,88 @@
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column            |
+|----------|--------------------------|-------------------|
+| AWS      | CUR                      | Product_region    |
+| GCP      | Big Query Billing Export | Location.location |
+| Azure    | EA, MCA                  | ResourceLocation  |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | Example Value                                                                   |
+|----------|--------------------------|---------------------------------------------------------------------------------|
+| AWS      | CUR                      | us-east-1 | eu-west-1 | us-gov-east-1                                          |
+| GCP      | Big Query Billing Export | northamerica-northeast1 | asia-southeast1 | us-central2 |  europe-west3 | us |
+| Azure    | Cost details             | Eastasia | AP East | WestUS | centralindia                                   |
+
+## Discussion / Scratch space:
+
+-   AWS provides one region (fixed set of values supported)
+
+-   Azure has two regions (metered region and the region where a product/resource is running)
+
+-   99% of the time the meter region is the same as resource region
+
+-   There are cases where multiple regions from the same provider or different providers may be specified in the provider specified region column
+
+-   Naming convention needs to be lower-case, or consider consistent naming convention
+
+-   Out of Scope for 0.5: TimeZone
+
+-   Azure: ResourceLocation (Where Azure resource resides),MeterRegion (how resource is billed on)
```

Comment:
This is covered on line 25. Consider removing one of them or making it a sub-bullet.